flexmeasures/cli/utils.py

```diff
@@ -301,3 +302,40 @@ def get_sensor_aliases(
     }
 
     return aliases
+
+
+def validate_color_cli(ctx, param, value):
+    """
+    Optional parameter validation
+
+    Validates that a given value is a valid hex color code.
+
+    Parameters:
+    :param ctx:     Click context.
+    :param param:   Click parameter. Hex value.
+    """
+
+    try:
+        validate_color_hex(value)
+    except ValueError as e:
+        click.secho(str(e), **MsgStyle.ERROR)
+        raise click.Abort()
+
+
+def validate_url_cli(ctx, param, value):
+    """
+    Optional parameter valdiation
```

Comment:
val*i*dation

Are you using Visual Studio? Then try [this](https://flexmeasures.readthedocs.io/en/latest/dev/setup-and-guidelines.html#using-visual-studio-including-spell-checking).